pkgs/shells/antigen/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  version = "2.2.1";
+  name = "antigen";
+
+  src = fetchurl {
+    url = "git.io/antigen";
```

Comment:
This source is not stable. Is there a different source, which does not break with every upstream update?